shared/overview/2.x.md

```diff
@@ -4,52 +4,55 @@
+| 2.46.1+rev3-dev | development | This version the development version of the above and should be used when you are developing a new application and want to use the fast [local mode][local-mode] workflow. This variant should never be used in production. |
 
-### Dev vs. Prod images
+### Development vs. Production images
 
-The Development images are recommended while getting started with {{ $names.os.lower }} and building a system.
-The dev images enable a number of useful features while developing, namely:
+__Warning:__ Development images have an exposed Docker socket and enable passwordless root SSH access and should never be used in production.
 
-* Passwordless SSH into {{ $names.os.lower }} on port `22222` as the root user. So one can do `ssh root@<DEVICE_IP> -p22222` and poke around to see how the system runs.
-* Docker socket exposed on port `2375`, which allows `{{ $names.company.lower }} push` / `build` / `deploy`
-  to do remote Docker builds on the target device (see [Deploy to your Fleet][deploy-to-fleet]).
+The development images are recommended while getting started with {{ $names.os.lower }} and building an application. The development images enable a number of useful features while developing, namely:
+
+* Passwordless [SSH access][ssh-host] into {{ $names.os.lower }} on port `22222` as the root user.
+* Docker socket exposed on port `2375`, which allows `{{ $names.company.lower }} push` / `build` / `deploy`, that enables remote Docker builds on the target device (see [Deploy to your Fleet][deploy-to-fleet]).
 * Getty console attached to tty1 and serial.
-* Capable of entering [local mode][local-mode-link] for rapid development of application containers locally.
+* Capable of entering [local mode][local-mode] for rapid development of application containers locally.
 
 __Note:__ Raspberry Pi devices don’t have Getty attached to serial.
 
-The production images have all of the above functionality disabled by default. In both forms of the OS we write logs to an 8 MB journald RAM
-buffer in order to avoid wear on the flash storage used by most of the supported boards. However, persistent logging can be enabled by setting
-the `"persistentLogging": true` key in the `config.json` file in the boot partition of the device. The logs can be accessed [via the host OS][ssh-host] at `/var/log/journal`.
+Production images disable root passwordless access, and an SSH key must be [added][config-json-ssh] to `config.json` to access a production image using a direct SSH connection. You may still access a production image by tunneling SSH through the {{ $names.company.lower }} VPN via the CLI (using `balena ssh <uuid>`) or the {{ $names.cloud.lower }} [web terminal][ssh-host]. To use SSH via the VPN, you need to have an SSH key configured on your development machine and [added][ssh-key-add] to the {{ $names.cloud.lower }} dashboard.
```

Comment:
```suggestion
Production images disable passwordless root access, and an SSH key must be [added][config-json-ssh] to `config.json` to access a production image using a direct SSH connection. You may still access a production image by tunneling SSH through the {{ $names.company.lower }} VPN via the CLI (using `balena ssh <uuid>`) or the {{ $names.cloud.lower }} [web terminal][ssh-host]. To use SSH via the VPN, you need to have an SSH key configured on your development machine and [added][ssh-key-add] to the {{ $names.cloud.lower }} dashboard.
```

Reply:
Took me about 5 mins to spot that change 😆

Reply:
heh, sorry :) i never understand why GH fails miserably sometimes with these simple diffs.